kaldb/src/main/java/com/slack/kaldb/writer/LogMessageWriterImpl.java

```diff
@@ -65,24 +61,14 @@ public class LogMessageWriterImpl implements MessageWriter {
         final Murron.MurronMessage murronMsg =
             murronMessageDeserializer.deserialize("", record.value());
         Trace.Span apiSpan = MurronLogFormatter.fromApiLog(murronMsg);
-        return SpanFormatter.toLogMessage(Trace.ListOfSpans.newBuilder().addSpans(apiSpan).build());
```

Comment:
I believe we have left this transformer because so many tests depend on it. So we can keep it. However this also always emits one span.